presets/tests/260-compshader-noise_lq.milk

```diff
@@ -0,0 +1,13 @@
+[preset00]
```

Comment:
So happy to see new test presets!

Reply:
Oh, does it really require this line in Milkdrop? The projectM parser just ignores it. :D

These presets were the easiest way to visualize the noise textures, before and after. I've also made a version that cyclces through the four individual color channels of the 2D noise texture plus RGB combined to make it clear the old implementation simply lacked a lot of random data. I can also add this if it's useful.

```
[preset00]
per_frame_1000=// spectrum vs pcm

fDecay=0
warp=0.000000
wave_a=0

comp_1=`shader_body
comp_2=`{
comp_3=`float t = time % 5;
comp_4=`if(t == 0) { ret = float3(tex2D(sampler_fw_noise_lq, uv).x); }
comp_5=`if(t == 1) { ret = float3(tex2D(sampler_fw_noise_lq, uv).y); }
comp_6=`if(t == 2) { ret = float3(tex2D(sampler_fw_noise_lq, uv).z); }
comp_7=`if(t == 3) { ret = float3(tex2D(sampler_fw_noise_lq, uv).w); }
comp_8=`if(t == 4) { ret = tex2D(sampler_fw_noise_lq, uv); }
comp_9=`}
```